burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
I don't think you need padding in the corners or at least the values are never used. This is because the filter will run in each 1d direction separately.